.github/workflows/build.yml

```diff
@@ -69,11 +69,35 @@ jobs:
       - name: Test postpack
         run: yarn postpack
 
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    needs: [test]
```

Comment:
Shouldn't we build and test in parallel, then trigger the release?

Reply:
You are totally right

Reply:
done, now testing and building needs lint (for energy saving purposes) and release needs build and test (they are run in parralele)